x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -554,28 +554,9 @@ export const nodesAndEdgelines: (state: DataState) => (
       processNodePositions: visibleProcessNodePositions,
       connectingEdgeLineSegments,
     };
-  }, aaBBEqualityCheck);
+  }, aabbModel.isEqual);
```

Comment:
New reselect package correctly induces the type here, so the helper function below is no longer necessary. Functionality is the same.